pinot-tools/src/main/java/org/apache/pinot/tools/PartialUpsertQuickStart.java

```diff
@@ -123,7 +67,10 @@ public void execute()
     printStatus(Color.CYAN, "Query : " + q2);
     printStatus(Color.YELLOW, prettyPrintResponse(runner.runQuery(q2)));
     printStatus(Color.GREEN, "***************************************************");
+  }
 
-    printStatus(Color.GREEN, "You can always go to http://localhost:9000 to play around in the query console");
+  @Override
+  public List<String> types() {
+    return Arrays.asList("PARTIAL-UPSERT", "PARTIAL_UPSERT");
```

Comment:
whats the difference between "PARTIAL-UPSERT", "PARTIAL_UPSERT"?

Reply:
This is used as QuickStart type when you start QuickStart using Pinot-admin script